bundles/org.openhab.binding.broadlink/README.md

```diff
@@ -0,0 +1,93 @@
+| pollingInterval     | Integer | 30            | The interval in seconds for polling the status of the device                      |
+| retries             | Integer | 1             | The number of re-attempts for a request before the device is considered `OFFLINE` |
+| ignoreFailedUpdates | Boolean | false         | If enabled, failed status requests won't put the device `OFFLINE`                       |
+| mapFilename         | String  | broadlink.map | The map file that contains remote codes to send via IR                            |
+
+> The `mapFilename` setting is applicable to the RM series of devices only.
+
+## Channels
+
+| Channel          | Supported Devices        | Type                 | Description                                     |
+|------------------|--------------------------|----------------------|-------------------------------------------------|
+| powerOn          | mp2, sp1, sp2, sp3, sp3s | Switch               | Power on/off for switches/strips                |
+| nightLight       | sp2, sp3, sp3s           | Switch               | Night light on/off                              |
+| temperature      | a1, rm2, rm4             | Number:Temperature   | Temperature                                     |
+| humidity         | a1, rm4                  | Number:Dimensionless | Air humidity percentage                         |
+| noise            | a1                       | String               | Noise level: `QUIET`/`NORMAL`/`NOISY`/`EXTREME` |
+| light            | a1                       | String               | Light level: `DARK`/`DIM`/`NORMAL`/`BRIGHT`     |
+| air              | a1                       | String               | Air quality: `PERFECT`/`GOOD`/`NORMAL`/`BAD`    |
+| s1powerOn        | mp1, mp1_1k3s2u          | Switch               | Socket 1 power                                  |
+| s2powerOn        | mp1, mp1_1k3s2u          | Switch               | Socket 2 power                                  |
+| s3powerOn        | mp1, mp1_1k3s2u          | Switch               | Socket 3 power                                  |
+| s4powerOn        | mp1                      | Switch               | Socket 4 power                                  |
+| usbPowerOn       | mp1_1k3s2u               | Switch               | USB power                                       |
+| powerConsumption | mp2, sp2,sp3s            | Number:Power         | Power consumption                               |
+| command          | rm, rm2, rm3, rm3q, rm4  | String               | IR Command code to transmit                     |
+
+## Map File
+
+The Broadlink RM family of devices can transmit IR codes.
+The map file contains a list of IR command codes to send via the device.
+The file uses the *Map Transformation* and is stored in the `<OPENHAB_CONF>/transform` folder.
+By default, the file name is `broadlink.map` but can be changed using the `mapFile` setting.
+
+Here is a map file example:
```

Comment:
Where do you get the map file from? Do you have to create it, or do you need to copy it from somewhere? Some further instructions for a new user would be great.